### ISSUE TITLE
Accessibility: add automated tests

### ISSUE THREAD

**Issue by biou** [goblindegook/littlefoot#468]
The test coverage of this library is good, but there is no automated accessibility tests. 
I recommend aXe as a solution designed to avoid false positives and aXe has bindings for Cypress. It cannot detect all accessibility issues but about 20% of them (still better than no tests)
It could be configured to check only for WCAG A and AA issues.
